### PR TITLE
Fixed incorrect message when deleting category

### LINKED ISSUE #4730: Incorrect confirmation message on deleting more than one category
The confirmation toast message should be in correct plain english when deleting more than one Category.

https://github.com/ushahidi/platform/assets/8624777/4307dbd2-6bc0-4f28-94e2-1647e26ba1ca

---
Created automatically from USH-976 (https://linear.app/ushahidi/issue/USH-976/incorrect-confirmation-message-on-deleting-more-than-one-category)

### PR DESCRIPTION
**Pull request by larrytamnjong**
Hello @Angamanga!

This pull request attempts to fix https://github.com/ushahidi/platform/issues/4730 where the delete confirmation message has awkward language when you select just one category to delete while using the bulk delete action. 

This particular issue is consistent across various sections; the same issue occurs for User, Survey, etc. I haven't addressed those yet, as I wanted to get this PR reviewed and if it is up to standard with the team once reviewed, I'll go ahead and implement the fix in other sections or make changes as you recommend.

**Before**

https://github.com/ushahidi/platform-client-mzima/assets/122117063/d21a34fc-a597-48cb-b4db-f94734bdb18f

https://github.com/ushahidi/platform-client-mzima/assets/122117063/764cceab-ebc2-4a77-96d5-b6292e05561a

**After**

https://github.com/ushahidi/platform-client-mzima/assets/122117063/3a1a1292-1252-4d3b-b5c6-5faff32651d0

https://github.com/ushahidi/platform-client-mzima/assets/122117063/96a46efd-9312-4812-a619-a9e4ee78eb1d

### CODE REVIEW

**Comment by Angamanga** [comment]
Thank you for your PR @larrytamnjong! @ushahidlee Could you please review?

File: apps/web-mzima-client/src/app/settings/categories/categories.component.ts [hunk 1]
```diff
@@ -62,14 +61,28 @@ export class CategoriesComponent {
...
+    const count = this.selectedCategories.length;
+    return {
+      title: this.getMessage('destroy_confirm', isBulk, count),
+      description: this.getMessage('destroy_confirm_desc', isBulk, count),
+      toast: this.getMessage('destroy_success', isBulk, count),
+    };
+  }
+
+  private getMessage(baseKey: string, isBulk: boolean = false, count?: number): string {
+    const prefix = isBulk ? 'bulk_' : '';
```
Comment (ushahidlee) [inline apps/web-mzima-client/src/app/settings/categories/categories.component.ts:85]: Could we not use count to establish prefix?
eg. const prefix = count > 1 ? 'bulk_' : '';

**Review by ushahidlee — APPROVED** [review]
Made a suggested change but overall looks fine

**Comment by larrytamnjong** [comment]
> Made a suggested change but overall looks fine

Hello @ushahidlee thanks a lot for the review. You're absolutely correct. Using the count variable to establish the prefix simplifies the logic and makes the code cleaner. I will make this change and re-submit for review. What do you think about me implementing this same solution to the User and Survey pages to fix same issue?

**Pull request merged**